src/lib/message/L1ToL2Message.ts

```diff
@@ -93,6 +93,9 @@ export abstract class L1ToL2Message {
    * creation of the retryable ticket.
    */
   public readonly retryableCreationId: string
+  // autoRedeemId & l2TxHash used for classic events
+  public readonly autoRedeemId: string | undefined
```

Comment:
I dont think these should be on the base class, since they are relevant to all types of L1ToL2Message

Reply:
In that case I guess we wouldn't be able to read `autoRedeemId ` and `l2TxHash` when having the message object returned? TypeScript will give us errors because we have no way of knowing if tx is classic beforehand (no block number when making the request).

Reply:
We dont appear to use these in the codebase, should they be used in `waitForStatus`? Since autoredeemid is only useful if you know if the message is classic or not the user should be checking whether the value is null, if they do this they could also cast to the classic message. This approach means the sdk doesnt support classic messages very nicely, but it still makes it possible to use the sdk for this if you know what you're doing. We can chat more about it in catchup?